docs/azure/migration/appcat/dotnet-cli.md

```diff
@@ -0,0 +1,94 @@
+    ![Screenshot of the analysis summary](./media/cli/cli-scan-finished.png)
+
+## Interpret the results
+
+For the purposes of explaining the results of the analysis, we're going to use the HTML report. The report presents its results in a dashboard format. In this section we'll describe the different sections of the dashboard and how to interpret them.
+
+### Dashboard view
+
+The main dashboard shows a summary of the results of the scan. It shows the number of projects scanned, the number of issues found, it estimates the amount of effort to fix the issues in story points, and it classifies the issues by severity from Mandatory to Informational.
+
+Here are the issue severity classifications:
+
+* **Mandatory** - the issue has to be resolved for the migration to be successful.
+* **Optional** - the issue discovered is a real issue and fixing it could improve the app after migration, however it is not blocking.
+* **Potential** - we are not sure if it is necessarily a blocking problem, but raised just in case.
+* **Informational** - the issue was raised only for informational purpose and is not required to be resolved.
+
+![Screenshot of the application and code assessment tool for .NET dashboard](./media/cli/cli-report-dashboard.png)
+
+The dashboard displays a graph of the category of each issue found. The categories can range from HTTP, to database, to scaling, and so on. The categories are based upon rules to identify issues.
+
+The **Summary** section of the dashboard contains several terms that are worth defining as you'll see them in other screens.
+
+* **Projects**: the number of projects scanned.
+* **Issues**: the number of unique encounters of a rule that may need to be addressed.
+* **Incidents**: the total number of occurences of all issues found.
+* **Story points**: the estimated effort to fix all of the issues found. This is a relative measure of effort and is not meant to be an exact estimate.
+
+### Projects view
+
+Click on the **Projects** link below the **Dashboard** on the left side of the report to see the number of issues, incidents, and the estimated effort to fix those issues by each project scanned.
+
+![Screenshot of the projects dashboard](./media/cli/cli-report-projects-overview.png)
+
+You can drill down to see the issues found in each project by clicking on the project name. This will show a screen similar to the overall dashboard but scoped to the selected project.
+
+![Screenshot of the a single project's dashboard](./media/cli/cli-report-project-dashboard-details.png)
+
+At the top of the project dashboard you'll find 3 tabs: **Dashboard**, **Components**, and **Issues**.
+
+Click on the **Components** tab to see which files the incidents of the issues identified reside in. You can drill down into the file to see the rule that triggered the incident, a description of the rule, the exact position in the code where the issue exists, and an estimation of the effort it will take to fix.
```

Comment:
```suggestion
Click on the **Components** tab to see which files the incidents of the issues identified reside in. You can drill down into the file to see the issues that triggered the incident, a description of the issue, the exact position in the code where the incidents exist, and an estimation of the effort it will take to fix the incidents.
```